types/node/ts3.2/node-tests.ts

```diff
@@ -57,6 +57,7 @@ import { BigIntStats, statSync, Stats } from 'fs';
 // FS Tests
 {
     const bigStats: BigIntStats = statSync('.', { bigint: true });
+    const bigIntStat: bigint = bigStats.atimeNs;
```

Comment:
It might be a good idea to also copy this change to `types/node/v12/ts3.2/node-tests.ts`.

Reply:
I'm out atm and can't really fix this for a while.
I'd like to get this merged as is otherwise you're welcome to create a followup PR.